lca-chimevc-stack/cloudformation-templates/chime-vc-call-transcriber.yaml

```diff
@@ -0,0 +1,293 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Transform: AWS::Serverless-2016-10-31
+
+Description: Amazon Transcribe Live Call Analytics with Agent Assist - Chime Voice Connector Call Transcriber Lambda
+
+Parameters:
+  AudioFilePrefix:
+    Type: String
+    Default: lca-audio-recordings/
+    Description: >-
+      The Amazon S3 prefix where the merged output audio files will be saved (must end in "/")
+
+  MonoAudioFilePrefix:
+    Type: String
+    Default: lca-mono-audio-recordings/
+    Description: >-
+      The Amazon S3 prefix where the mono output audio files will be saved (must end in "/")
+
+  DynamoDbExpirationInDays:
+    Type: Number
+    Default: 90
+    Description: >-
+      Number of days set in the time to live of event data stored in the DynamoDB table
+    
+  KinesisDataStreamName:
+    Type: String
+    Description: >-
+      Name of Kinesis Data Stream to publish events to
+  KinesisDataStreamArn:
+    Type: String
+    Description: >-
+      Arn of Kinesis Data Stream to publish events to
+  EventSourcingTableName:
```

Comment:
We had talked about giving this function it's own DynamoDB table rather than using the same table used by AppSync in the AISTACK. Did you find a reason not to do that?